docs/requester_pays.rst

```diff
@@ -0,0 +1,195 @@
+=======================================================================================
+Accessing Public Cloud Data with Requester Pays
+=======================================================================================
+
+The data we're publishing in the PUDL Catalog is publicly accessible and distributed
+under the permissive `CC-BY-4.0 <https://creativecommons.org/licenses/by/4.0>`__
+license. Catalyst covers the cost of storing the data in Google cloud storage buckets.
+However, there are also fees incurred when data leaves the Google Cloud Platform (GCP).
+Depending where you're downloading from, it costs $0.10-0.20 (USD) per GB.  In order to
+share large amounts of public data without risking unexpectedly large data egress
+charges due to someone maliciously or accidentally downloading a large volume of data,
+we've set our cloud storage to use `requester pays
+<https://cloud.google.com/storage/docs/requester-pays>`__.
+
+"Requester pays" means the person downloading the data is responsible for those costs
+instead. As a user paying the data egress charges associated with your data usage is
+cheap, but as a data provider covering the data egress charges for all of your users can
+quickly get expensive! Downloading all of the EPA CEMS, FERC 1, PUDL, and US Census data
+we're publishing from North America should cost about $0.75, but if we had 1000
+downloads in a month that would cost of $750. The PUDL Intake catalog is also set up to
```

Comment:
"would cost of" --> "would cost upwards of" ?

Reply:
Okay, simplified.